app/graphql/types/query_type.rb

```diff
@@ -0,0 +1,13 @@
+module Types
+  class QueryType < Types::BaseObject
+    # Add `node(id: ID!) and `nodes(ids: [ID!]!)`
+    include GraphQL::Types::Relay::HasNodeField
+    include GraphQL::Types::Relay::HasNodesField
+
+    field :projects, resolver: Resolvers::Projects
+    field :project, resolver: Resolvers::Project
+
+    field :tasks, resolver: Resolvers::Tasks
```

Comment:
I wouldn't include tasks to query. I'd make it accessible only via project

Reply:
@DmitryBarskov How can I do this more correctly?